pkg/install/verify.go

```diff
@@ -58,7 +56,7 @@ func Verify(cl *lokomotive.Cluster) error {
 		return ns.Ready(), nil // Retry if not ready
 	})
 	if nsErr != nil {
-		return errors.Wrapf(nsErr, "error determining node status within the allowed time")
+		return fmt.Errorf("determining node status within allowed time: %w", nsErr)
```

Comment:
I find "determining node status within allowed time" strange. IIUC this error is the *last* error received from `GetNodeStatus()` after giving up due to timer expiration. I'd consider using the following phrasing:

`return fmt.Errorf("waiting for nodes: %w", nsErr)`

Reply:
Changed as you suggested.